composer.json

```diff
@@ -3,14 +3,15 @@
 	"description" : "The most popular FOSS online survey tool on the web",
 	"homepage" : "https://www.limesurvey.org/",
 	"license" : "GPL 2.0",
+	"type": "limesurvey",
```

Comment:
https://getcomposer.org/doc/04-schema.md#type

Reply:
?

Reply:
It was a mistake... sorry about that